.circleci/config.yml

```diff
@@ -191,4 +190,18 @@ jobs:
           command: |
             pip install --upgrade pip
             pip install "."
-            bash script/test/package.sh
\ No newline at end of file
+            bash script/test/package.sh
+
+  Docker:
+    resource_class: large
+    docker:
+      - image: "cimg/python:3.10"  
```

Comment:
What is diff between this ci image and the image pulled by Dockerfile?

Reply:
Will try removing it.